compiler/modules/types/core.bal

```diff
@@ -214,6 +214,12 @@ public type ListFiller readonly & record {|
     Filler[] memberFillers;
 |};
 
+public function listFillerSemType(Env env, ListFiller filler) returns SemType {
```

Comment:
Needs a comment saying what it does

Reply:
This seems wrong.

Reply:
Can we solve this problem by including the semType equivalent of atomic in ListFiller

Reply:
Have listAtomicTypeToSemType in list.bal.